aptos-move/vm-genesis/src/lib.rs

```diff
@@ -350,7 +350,16 @@ fn publish_stdlib(session: &mut SessionExt<impl MoveResolver>, stdlib: Modules)
 }
 
 /// Trigger a reconfiguration. This emits an event that will be passed along to the storage layer.
-fn reconfigure(session: &mut SessionExt<impl MoveResolver>) {
+fn emit_new_block_and_epoch_event(session: &mut SessionExt<impl MoveResolver>) {
+    exec_function(
+        session,
+        "block",
+        "emit_genesis_block_event",
+        vec![],
+        serialize_values(&vec![MoveValue::Signer(
+            account_config::reserved_vm_address(),
```

Comment:
What's the policy on using vm address (0x0) vs aptos framework (0x1)? We're using these both in genesis.

Reply:
I think it's to separate privileges, so even aptos framework is not able to emit a block

Reply:
Makes sense